src/Microsoft.AspNetCore.Hosting/Internal/HostingApplication.cs

```diff
@@ -9,11 +9,17 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Features;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
 
 namespace Microsoft.AspNetCore.Hosting.Internal
 {
     public class HostingApplication : IHttpApplication<HostingApplication.Context>
     {
+#if !NET451
+        private const string ActivityName = "Microsoft.AspNetCore.Hosting.Activity";
```

Comment:
Don't call it 'Activity'.  It should be the name what the activity is.  Thus logically is should be 

    Microsoft.AspNetCore.Hosting.IncommingHttpRequest

But I would shorten it to 

    Microsoft.AspNetCore.Hosting.HttpReqIn